HW_03.11/Asynchronous/Asynchronous/Program.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Asynchronous
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            ComeBackHome();
+            var preparingSauce = PrepareSauce();
+            Sauce sauce = await preparingSauce;
```

Comment:
Sauce sauce = await PrepareSauce();

since these two lines are one right after another it's ok to await right away